Sources/Purchasing/StoreKitAbstractions/StoreProduct.swift

```diff
@@ -84,7 +84,7 @@ public typealias SK2Product = StoreKit.Product
     @available(iOS 12.0, macCatalyst 13.0, tvOS 12.0, macOS 10.14, watchOS 6.2, *)
     @objc public var subscriptionGroupIdentifier: String? { self.product.subscriptionGroupIdentifier}
 
-    @objc public var priceFormatter: NumberFormatter? { self.product.priceFormatter }
+    @objc public var priceFormatter: NumberFormatter { self.product.priceFormatter }
```

Comment:
does this make it a breaking change?

Reply:
Yeah which is why I made it a `feat`. But I had to revert this to fix Xcode 13.x compilation 🤷🏻‍♂️ 
Won't matter in the long term since you can't submit apps with that anymore, but we need to keep support for it in CI until I finish #2421.